components/style/gecko_bindings/test_bindings.rs

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
```

Comment:
You'll need to include this module somewhere, otherwise it wont be compiled. And you'll need to generate the relevant `use` statements too.

A line like:

```rust
mod test_bindings;
```

in `gecko_bindings/lib.rs` would suffice for the first part.